ProjectCoimbra.UWP/Project.Coimbra/Pages/GamePage.xaml.cs

```diff
@@ -214,5 +214,37 @@ private void Timer_Tick(object sender, object e)
                 }
             }
         }
+
+        /// Only pause the playback in SOLO mode
+        private async void Pause_Click(object sender, RoutedEventArgs e)
+        {
+            if (UserData.GameMode == UserData.Mode.Solo)
+            {
+                midiEngine.Pause();
+            }
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = "Game paused",
+                Content = "Do you want to continue the game?",
+                PrimaryButtonText = "Continue",
+                CloseButtonText = "Exit to Game Mode",
+                DefaultButton = ContentDialogButton.Primary
+            };
+
+            var result = await dialog.ShowAsync();
+            
+            if (result != ContentDialogResult.Primary)
```

Comment:
What does it mean if we're in this `if` block? How would this occur?

Can you add a comment to clarify?

Reply:
L#234 is the result of dialog, which `Primary: Resume`  and `Close: Exit to Game mode``
If user does not select Primary (that means he wants to exit the game), then the MidiEngine dispose the playback, and exit the game

Reply:
Could you add a comment with this detail to the code, so that people working with the code in the future can understand this? Thanks